events/iisc_start_events.txt

```diff
@@ -1,4 +1,19 @@
 namespace = iisc_start
+event = {
+	id = iisc_start.0
+	is_triggered_only = yes
+	hide_window = yes
+
+	immediate = {
+		set_global_flag = iisc_corp_spawning_all
+		set_global_flag = iisc_mining_stations_all
+		set_global_flag = iisc_research_stations_all
+		set_global_flag = iisc_corp_surveying_all
+		set_global_flag = iisc_corp_convoys_all
+		set_global_flag = iisc_hq_development_all
+	}
+}
+
```

Comment:
This is triggered by game start, but we may also need to set these/have these triggered when the mod is added to an ongoing game.

Reply:
Added!